fvm/fvm.go

```diff
@@ -27,11 +27,21 @@ func NewInterpreterRuntime(config runtime.Config) runtime.Runtime {
 
 // A VirtualMachine augments the Cadence runtime with Flow host functionality.
 type VirtualMachine struct {
-	// TODO(patrick): move this into ReusableCadenceRuntime
-	Runtime runtime.Runtime
+	Runtime runtime.Runtime // DEPRECATED.  DO NOT USE.
 }
 
-// NewVirtualMachine creates a new virtual machine instance with the provided runtime.
+func NewVM() *VirtualMachine {
```

Comment:
NOTE: I had to create this second function because of our ridiculous integration test setup.  We basically need 2-3 sporks in order to deprecate this function